src/pages/ProductListing.js

```diff
@@ -1,10 +1,46 @@
-import React from 'react'
-
+import React from "react";
+import "../App.css";
+import { FilterSection } from "../component/index";
+import { useProduct } from "../context/product-context";
+import { FaHeart } from "react-icons/fa";
 function ProductListing() {
-    return (
-        <div>
-            <h1>ProductListing Page</h1>
-        </div>
-    )
+  const { filteredProdList } = useProduct();
+  return (
+    <div className="prodListing-main">
+      <FilterSection />
+      <div className="prodList-div gap-m">
+        {filteredProdList.map((prod) => {
+          return (
+            <div class="card ">
```

Comment:
Extract the card into a new component as you're gonna be reusing this

Reply:
Done Extract it into productCart component